docs-mslearn/framework/understand/ingestion.md

```diff
@@ -108,6 +108,21 @@ At this point, you have a data pipeline and are ingesting data into a central da
 
 <br>
 
+## Data Ingestion Metrics
+
+| **Category** | **Definition** | **KPI** |
+|----------|-----------|-----|
+| Data Completeness | Measures the extent to which all required data fields are present in the dataset and tracks the overall data completeness trend over a specified period.| Percentage of data fields that are complete and the overall data completeness over time. |
+| Data Quality Performance | Measures the percentage of successful data quality checks and the total number of data quality checks conducted within a specified period. | Number of data quality checks conducted and the percentage of successful data quality checks . |
```

Comment:
We should probably avoid "performance" since that sounds like a measurement of time. "Data quality" is probably good enough (especially as a category). If this is intended to be a KPI name, then maybe something like "pass rate". I'm sure there's something better than that.
```suggestion
| Data quality | Measures the percentage of successful data quality checks and the total number of data quality checks conducted within a specified period. | Number of data quality checks conducted and the percentage of successful data quality checks . |
```